frontend/front/src/api/apiSlice.js

```diff
@@ -29,6 +29,17 @@ export const apiSlice = createApi({
         ...result.map(({ id }) => ({ type: "Home", id })),
       ],
     }),
+    getUnassignedHomes: builder.query({
+      query: () => "/homes",
+      transformResponse: (res) =>
+        res
+          ? res.filter((home) => home.assignment_id === null).sort(sortById)
```

Comment:
You should be able to ditch the `.filter` here and just do
```
query: () => "/homes?assignment_id=null"
```
do delegate the filter to the back end!

Reply:
hmmm, this doesn't return a response for me. However, this does:
`query: () => "/homes?assignment_id"`

Reply:
Oh weird, it worked for me (with curl anyway)

Reply:
I tried in the app and also Postman with `localhost:3000/homes?assignment_id=null` and just get an empty array. Not sure what's going on there.